package.json

```diff
@@ -8,14 +8,11 @@
     "url": "https://github.com/zowe/vscode-extension-for-zowe"
   },
   "private": true,
-  "workspaces": [
-    "packages/*"
-  ],
   "engines": {
     "vscode": "^1.53.2"
   },
   "dependencies": {
-    "@zowe/cli": "7.16.6",
+    "@zowe/cli": "7.16.2",
```

Comment:
Why is the dependency version of Zowe CLI decreasing here?

Reply:
Might've been from the cherry-picked old commits, I can go ahead and bump up the version, thanks for catching that.